src/app/src/__tests__/util.ga.tests.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-env jest */
+/* eslint-disable object-shorthand */
+/* eslint-disable func-names */
+/* eslint-disable space-before-function-paren */
+
+import {
+    userHasAcceptedOrRejectedGATracking,
+    userHasAcceptedGATracking,
+    userHasRejectedGATracking,
+    rejectGATracking,
+    acceptGATrackingAndStartTracking,
+    clearGATrackingDecision,
+    createGADisableKey,
+} from '../util/util.ga.js';
+
+beforeEach(() => {
+    const window = {};
+
+    window.localStorage = {
+        store: {},
+        getItem: function(key) {
+            return this.store[key] || null;
+        },
+        setItem: function(key, value) {
+            this.store[key] = value.toString();
+        },
+        removeItem: function(key) {
+            delete this.store[key];
+        },
+        clear: function() {
+            this.store = {};
+        },
+    };
+
+    window.ENVIRONMENT = {
+        REACT_APP_GOOGLE_ANALYTICS_KEY: 'GA_KEY',
+    };
+
+    global.window = window;
+    jest.spyOn(global.window.localStorage, 'setItem');
+});
+
+afterEach(() => {
```

Comment:
Should we be restoring `global.window` once we're finished?

Reply:
I accidentally marked this as resolved but I don't think so. Given that window.localStorage is also re-assigned, I think setting it to `null` here kind of doesn't matter anymore for the tests.